### PR TITLE
Add suite.Run(...) to all table driven tests

### PR DESCRIPTION
**Pull request by AlpinYukseloglu**
Closes: #1990

## What is the purpose of the change

Our usage of `suite.Run(...)` is currently very inconsistent across tests in our codebase. This PR adds it to the table driven tests that don't have it already and make it a testing standard to use it in future tests we create.

This mainly serves to make our tests structure more consistent while also making it a bit easier to debug issues in the future.

## Brief Changelog

- Add `suite.Run(...)` or its relevant variant to table driven module tests that don't already use it

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (no)
  - How is the feature or change documented? (not documented)

### CODE REVIEW

**Comment by stackman27** [comment]
I think these tests may've been missed; `multihop_test.go` `swap_test.go` and `msg_server_test.go` in lockup and superfluid keeper

File: tests/e2e/initialization/init_test.go [hunk 1]
```diff
@@ -16,11 +16,9 @@ import (
 
 const forkHeight = 10
 
-var (
-	expectedConfigFiles = []string{
-		"app.toml", "config.toml", "genesis.json", "node_key.json", "priv_validator_key.json",
-	}
-)
+var expectedConfigFiles = []string{
+	"app.toml", "config.toml", "genesis.json", "node_key.json", "priv_validator_key.json",
```
Comment (alexanderbez) [inline tests/e2e/initialization/init_test.go:20]: nit: can we make these multi-line please?

Comment (AlpinYukseloglu) [reply tests/e2e/initialization/init_test.go:20]: I believe this was done automatically through `make format` - I have no preference so happy to change back in this PR but just fyi

**Review by alexanderbez — APPROVED** [review]

**Review by p0mvn — APPROVED** [review]

**Comment by p0mvn** [comment]
@AlpinYukseloglu were only gamm / txfees tests missing these? Just making sure that we don't need follow-up issues for other modules

**Pull request merged**